src/game/shared/neo/neo_juggernaut.h

```diff
@@ -24,6 +24,8 @@ class CNEO_Juggernaut : public CBaseAnimating
 	bool m_bPostDeath = false;
 #endif
 
+	virtual unsigned int PhysicsSolidMaskForEntity(void) const { return MASK_PLAYERSOLID; }
```

Comment:
Nit: Would prefer if we mark this virtual `override`. And maybe `final` as well, as an optimization hint?

The `(void)` parameter is also redundant.

```diff
-       virtual unsigned int PhysicsSolidMaskForEntity(void) const { return MASK_PLAYERSOLID; }
+       virtual unsigned int PhysicsSolidMaskForEntity() const final override { return MASK_PLAYERSOLID; }
```